packages/search-ui/src/Filter/TabFilter.tsx

```diff
@@ -44,15 +45,15 @@ const TabFilter = (props: Omit<TabFilterProps, 'type'>) => {
           className={customClassNames.filter?.tabs?.tab}
           selectedClassName={customClassNames.filter?.tabs?.selectedTab}
         >
-          {t('all')}
+          {formatLabel('all', { textTransform, t })}
```

Comment:
![Screen Shot 2021-04-15 at 12 59 12](https://user-images.githubusercontent.com/25856620/114821216-6eb20280-9dea-11eb-85dd-ddb1b68080d8.png)
![Screen Shot 2021-04-15 at 13 03 10](https://user-images.githubusercontent.com/25856620/114821569-f1d35880-9dea-11eb-9483-c6d8b793ece8.png)
Difference between [new change](https://deploy-preview-461--sajari-react-sdk-v3.netlify.app/search-ui/filter#tabs) and [production](https://react.docs.sajari.com/search-ui/filter#tabs)

I think this should be
```suggestion
          {formatLabel(t('all'), { textTransform })}
```

Reply:
Fixed but we still need to pass `t` in.